addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -1101,7 +1104,18 @@ QUnit.module('basic_fields', {
                     });
                 }
                 if (route === "/web/dataset/call_kw/res.lang/get_installed") {
-                    return Promise.resolve([["en_US"], ["fr_BE"]]);
+                    return Promise.resolve([["en_US", "English"], ["fr_BE", "French (Belgium)"]]);
+                }
+                if (args.method === "search_read" && args.model == "ir.translation") {
+                    return Promise.resolve([
+                        {lang: 'en_US', src: 'yop', value: 'yop', id: 42},
+                        {lang: 'fr_BE', src: 'yop', value: 'valeur français', id: 43}
+                    ]);
+                }
+                if (args.method === "write" && args.model == "ir.translation") {
+                    assert.deepEqual(args.args[1], {value: "english value"},
+                        "Wrong update on translation");
```

Comment:
assert messages are actually displayed whether it is a success or not.
So the formulation is rather a positive one:
Here something like "The new translation value should be written"